app/actions/views/select_server.js

```diff
@@ -12,6 +13,24 @@ export function handleServerUrlChanged(serverUrl) {
     };
 }
 
+export function handleLoginOptions() {
+    return async (dispatch, getState) => {
+        const {config, license} = getState().entities.general;
+
+        let options = 0;
+        if (config.EnableSaml === 'true' && license.IsLicensed === 'true' && license.SAML === 'true') {
+            options++;
```

Comment:
I think @thomchop's method matches the webapp better. We frequently do stuff like
```javascript
const gitlabEnabled = ...;
const ldapEnabled = ...;
const samlEnabled = ...;

if (gitlabEnabled || ldapEnabled || samlEnabled) {
    ...
}
```

And I also agree with `+=` over `++` outside of loop counters